File: API/Features/ShipOwners/Dtos/ShipOwnerReadDto.cs
namespace API.Features.ShipOwners {

    public class ShipOwnerReadDto {

        public int Id { get; set; }
        public string Description { get; set; }
        public string Profession { get; set; }
        public string Address { get; set; }
        public string TaxNo { get; set; }
        public string City { get; set; }
        public string Phones { get; set; }
        public string Email { get; set; }
        public bool IsActive { get; set; }

    }

}

File: FrontEnd/src/app/features/shipOwners/classes/dtos/shipOwner-read-dto.ts
import { Metadata } from 'src/app/shared/classes/metadata'

export interface ShipOwnerReadDto extends Metadata {

    id: number
    description: string
    profession: string
    address: string
    taxNo: string
    city: string
    phones: string
    email: string
    isActive: boolean
    user: string
    lastUpdate: string

}
